src/test/tests/scanUtils.test.ts

```diff
@@ -0,0 +1,88 @@
+import { assert, expect } from 'chai';
+import { ScanUtils } from '../../main/utils/scanUtils';
+import sinon from 'sinon';
+import * as fs from 'fs';
+import * as path from 'path';
+import { isWindows } from './utils/utils.test';
+
+describe('ScanUtils', () => {
+    describe('executeCmdAsync', () => {
+        let clearIntervalSpy: sinon.SinonSpy;
+        beforeEach(() => {
+            clearIntervalSpy = sinon.spy(global, 'clearInterval');
+        });
+
+        afterEach(() => {
+            clearIntervalSpy.restore();
+        });
+
+        const dummyCheckError: () => void = () => {
+            return;
+        };
+
+        it('should execute a command successfully', async () => {
```

Comment:
Let's use the same project conventions for the test names, for example in this case:
```suggestion
        it('Execute a command successfully', async () => {
```